awscli/topics/config-vars.rst

```diff
@@ -476,6 +478,66 @@ To set encoding used for text files different from the locale, you can use
 with default encoding ``CP1252``, setting ``AWS_CLI_FILE_ENCODING=UTF-8`` would
 make CLI ignore locale encoding and open text files using ``UTF-8``.
 
+Pager
+-----
+
+The AWS CLI uses a pager for output data that does not fit on the screen.
+
+On Linux/MacOS, ``less`` is used as the default pager. On Windows,
+the default is ``more``.
+
+Configuring pager
+^^^^^^^^^^^^^^^^^^
+
+You can override the default pager with the following configuration
+options. These are in order of precedence:
+
+    ``AWS_PAGER`` environment variable
+
+    ``cli_pager`` shared config variable
+
+    ``PAGER`` environment variable
+
+If you set any of the configuration options to an empty string
+(e.g. ``AWS_PAGER=""``) or use ``--no-cli-pager`` option in the command line the
+AWS CLI will not send the output to a pager.
+
+Examples
+""""""""
+
+To disable the pager for ``default`` profile:
```

Comment:
We should make these examples use code blocks. You should be able to do that by using double colons (e.g. `::`) instead of a single colon for each example.